examples/nas/darts/model.py

```diff
@@ -48,18 +50,19 @@ def __init__(self, node_id, num_prev_nodes, channels, num_downsample_connect, dr
                         ops.DilConv(channels, channels, 3, stride, 2, 2, affine=False),
                         ops.DilConv(channels, channels, 5, stride, 4, 2, affine=False),
                     ],
-                    key="{}_p{}".format(node_id, i)))
-        self.drop_path = ops.DropPath_(drop_path_prob)
+                    key=choice_keys[-1]))
+        self.drop_path = ops.DropPath_()
+        self.input_switch = mutables.InputChoice(choose_from=choice_keys, n_chosen=2, key="{}_switch".format(node_id))
 
     def forward(self, prev_nodes):
         assert len(self.ops) == len(prev_nodes)
         out = [op(node) for op, node in zip(self.ops, prev_nodes)]
-        return sum(self.drop_path(o) for o in out if o is not None)
+        return self.input_switch(out)
```

Comment:
so `out` is not a dict?

Reply:
It can be a dict or a list. It's a list here for convenience.